packages/eslint-plugin/src/configs/react-config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     ],
   },
   rules: {
+    'react-compiler/react-compiler': ['error'],
```

Comment:
do we really want/need to have this for legacy codebase ? v8 etc 

we should aim this on v9 only -> move these rule definitions to `configs/react.js` 

thoughts?

Reply:
Yeah, good call!